scripts/keys.coffee

```diff
@@ -10,95 +10,309 @@
 #	bot xyz has keys : sets xyz as the holder of keys
 #
 # Examples:
-#   :bot who has keys
-#   :bot i have keys
-#   :bot i dont have keys
-#	:bot who has keys
-#	:bot ravi has keys
+#   :bot who has <color> keys
+#   :bot i have <color> keys
+#   :bot i dont have <color> keys
+#	:bot who has <color> keys
+#	:bot ravi has <color> keys
 #
 # Author:
 #   Punit Dhoot (@pdhoot)
-#   Developer at SDSLabs (@sdslabs)
+#   Developer at SDSLabs (@sdslabs
+#  Edit:
+#   prakhar gupta
+
 
 module.exports = (robot)->
 
-	key = ()->
-		Key = robot.brain.get("key") or ""
-		robot.brain.set("key" ,Key)
-		Key	
+	keyRed = ()->
+		Key1 = robot.brain.get("keyRed") or ""
+		robot.brain.set("keyRed",Key1)
+		Key1	
+	keyBlue = ()->
+		Key1 = robot.brain.get("keyBlue") or ""
+		robot.brain.set("keyBlue",Key2)
+		Key2	
+	keyYellow = ()->
+		Key1 = robot.brain.get("keyYellow") or ""
+		robot.brain.set("keyYellow",Key3)
+		Key3	
+	keyGreen = ()->
+		Key1 = robot.brain.get("keyGreen") or ""
+		robot.brain.set("keyGreen",Key4)
+		Key4	
+	keyOrange = ()->
+		Key1 = robot.brain.get("keyOrange") or ""
+		robot.brain.set("keyOrange",Key5)
+		Key5	
 
-	
-	robot.respond /i have (a key|the key|key|keys)/i, (msg)->
+    #1this section belongs to "i have keys"
```

Comment:
Remove "#1", "#2" etc from the start of each comment.

Reply:
@aseem09 done